src/adapters/MeetingsJSONAdapter.js

```diff
@@ -131,6 +139,27 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     return stream;
   }
 
+  /**
+   * Returns a MediaStream object obtained from local user media.
+   * @returns {MediaStream}
+   */
+  async getRemoteVideo() {
+    const constraints = {
+      video: true,
+      audio: false,
+    };
+    let stream;
+
+    try {
+      stream = await navigator.mediaDevices.getUserMedia(constraints);
+    } catch (reason) {
+      // eslint-disable-next-line no-console
+      console.error('Meetings JSON adapter can not display the user local stream', reason);
```

Comment:
```suggestion
      console.error('Meetings JSON adapter can not display the user remote stream', reason);
```
Technically not accurate but this is supposed to represent "remote media"

Reply:
it's just a copy-paste to try and get storybook mocking working, will fix